homeassistant/components/statistics/sensor.py

```diff
@@ -393,13 +395,12 @@ async def async_start_preview(
         await self._async_stats_sensor_startup()
         return self._call_on_remove_callbacks
 
-    @callback
-    def _async_stats_sensor_state_listener(
+    def _async_handle_new_state(
         self,
-        event: Event[EventStateChangedData],
+        reported_state: Any,
```

Comment:
Please don't collapse this to `Any`,  it's `State | None` https://github.com/home-assistant/core/blob/e7917e46ed6e784c5b18508ad483cf78fbfb191e/homeassistant/core.py#L154-L176

Reply:
that wasn't meant to stay, I oversaw it when checking in